share/gpodder/extensions/subscription_stats.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+# Show publishing statistics for subscriptions.
+# Released under the same license terms as gPodder itself.
+# 2023/11/24 - Nuno Dias <Nuno.Dias+gpodder@gmail.com>
+# Add Last Fedd and Episode updates
+
+import time
+from time import strftime, localtime
+
+import datetime
+from datetime import datetime
+
+import gpodder
+
+import gi  # isort:skip
+gi.require_version('Gtk', '3.0')  # isort:skip
+from gi.repository import Gtk, Pango  # isort:skip
+
+_ = gpodder.gettext
+
+__title__ = _('Subscription Statistics')
+__description__ = _('Show publishing statistics for subscriptions.')
+__only_for__ = 'gtk'
+__authors__ = 'Brand Huntsman <http://qzx.com/mail/>'
+
+
+class gPodderExtension:
+    def __init__(self, container):
+        self.container = container
+
+    def on_ui_object_available(self, name, ui_object):
+        if name == 'gpodder-gtk':
+            self.gpodder = ui_object
+
+    def on_create_menu(self):
+        # extras menu
+        return [(_("Subscription Statistics"), self.open_dialog)]
+
+    def add_page(self, notebook, type, channels):
+        scrolled = Gtk.ScrolledWindow()
+        scrolled.set_policy(Gtk.PolicyType.NEVER, Gtk.PolicyType.AUTOMATIC)
```

Comment:
The window width can be resized pretty small now, so the NEVER should be changed to AUTOMATIC, or remove the entire line since both default to automatic. However, something is causing the horizontal scrollbar to appear when the dialog is opened, and resizing narrower or wider hides it. Anyone know why?

Should we also set a minimum width for the channel name column? The name can be unusably short when resized to minimum width, more so for a paused channel.

Reply:
`channelcell.set_property('width-chars', 20)` makes the column at least 20 characters wide before ellipsizing it. This only matters if we decide to enable the horizontal scrollbar, but it would be better to just not ellipsize it, and then the width-char property isn't needed.

I don't care if we keep it as is (and not need to worry about the broken scrollbar), or enable the scrollbar and get rid of the ellipsis so the full name displays. Anyone with a narrow window might also have a narrow gpodder window and channel names in the podcast and episode lists would be ellipsized.

@tpikonen Do you have a preference?

Reply:
I've made some more tests and I think the best solution is to get rid of the ellipsis, that was introduced when the Channel was not the last column, so if the final design in the columns is "Updated | Days | Channel" we don't have anymore the problems that led to the introduction of the ellipsis.